<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCalc v2.0 kizzywyn</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="5" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">QWidget {
/*background-color: rgb(8, 1, 18);*/
background-color: rgb(16, 16, 35);
border: none;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageMenu">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>93</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QWidget" name="widgetMenu" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>300</height>
           </size>
          </property>
          <property name="font">
           <font/>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget {
background:none;
}

QPushButton {
font: 20pt &quot;Kode Mono&quot;;
color: rgb(255, 210, 255);

border-radius: 6px;

border-style: solid;
border-width: 2px;

/*
rgb(240, 179, 61);
rgb(123, 120, 195);
border-color: rgb(123, 120, 195);
*/
background-color: rgb(100, 97, 184);
}

QPushButton:hover {
background-color: rgb(169, 167, 215);
}

QPushButton:pressed {
background-color: rgb(25, 24, 53);
}

</string>
          </property>
          <widget class="QPushButton" name="pushButtonToCalculator">
           <property name="geometry">
            <rect>
             <x>19</x>
             <y>20</y>
             <width>161</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Calculator</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonToCredit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>80</y>
             <width>161</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Credit</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonToDeposit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>140</y>
             <width>161</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Deposit</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonExit">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>220</y>
             <width>161</width>
             <height>51</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgb(147, 11, 18);
}

QPushButton:hover {
background-color: rgb(239, 42, 51);
}

QPushButton:pressed {
background-color: rgb(71, 5, 9);
}
</string>
           </property>
           <property name="text">
            <string>Quit</string>
           </property>
           <property name="icon">
            <iconset resource="../misc/resource.qrc">
             <normaloff>:/icons/exit_to_app_yellow.svg</normaloff>:/icons/exit_to_app_yellow.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>93</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>375</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>375</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="Calculator" name="pageCalculator"/>
      <widget class="CreditCalculator" name="pageCredit"/>
      <widget class="DepositCalculator" name="pageDeposit"/>
     </widget>
    </item>
    <item row="2" column="0" rowspan="3">
     <widget class="QWidget" name="widgetTopMenu" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
background-color:rgb(151, 149, 208);
/*background-color: rgb(65, 8, 145);*/
/*background-color: rgb(8, 1, 18);*/
border-bottom: 0px solid;
border-color: rgb(30, 28, 27);
}

QPushButton {
font: 13pt &quot;Anta&quot;;
color: black;
border: none;
background: none;
}

QPushButton:hover {
color: rgb(255, 210, 255);
}

QPushButton:pressed {
background-color: rgb(255, 210, 255);
color: black;
border-radius: 4px;
border-width: 0px;
border-style: solid;
border-color: transparent;
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButtonCredit">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	image: url(:/icons/payments_black.svg);
}

QPushButton:hover {
image: url(:/icons/payments_yellow.svg);
}

QPushButton:pressed {
image: url(:/icons/payments_black.svg);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonMenu">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
image: url(:/icons/menu_black.svg);
}

QPushButton:hover {
image: url(:/icons/menu_yellow.svg);
}

QPushButton:pressed {
image: url(:/icons/menu_black.svg);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonCalculator">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	image: url(:/icons/calculate_black.svg);
}

QPushButton:hover {
image: url(:/icons/calculate_yellow.svg);
}

QPushButton:pressed {
image: url(:/icons/calculate_black.svg);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButtonDeposit">
         <property name="minimumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	image: url(:/icons/savings_black.svg);
}

QPushButton:hover {
image: url(:/icons/savings_yellow.svg);
}

QPushButton:pressed {
image: url(:/icons/savings_black.svg);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>17</width>
           <height>17</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCalculator">
   <property name="text">
    <string>Calculator</string>
   </property>
  </action>
  <action name="actionCredit">
   <property name="text">
    <string>Credit</string>
   </property>
  </action>
  <action name="actionDeposit">
   <property name="text">
    <string>Deposit</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Calculator</class>
   <extends>QWidget</extends>
   <header>calculator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreditCalculator</class>
   <extends>QWidget</extends>
   <header>creditcalculator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DepositCalculator</class>
   <extends>QWidget</extends>
   <header>depositcalculator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../misc/resource.qrc"/>
 </resources>
 <connections/>
</ui>
